cachediff.py

```diff
@@ -47,6 +47,39 @@ def has_virtual_address(self, address):
         return address in self.assembly_instructions
 
 
+def init_file(path):
+    '''
+    @return a sorted list of HighLine objects from the given filename
+    NOTE : the logic can be made simpler using proper REGEX
+    '''
+    import re
+    f = open('test_file_dump.dump','r')
```

Comment:
Remember that file argument will depend on the file. so it does not make sense to hardcode the file path just to make the test cases pass.